mlprimitives/jsons/sklearn.preprocessing.RobustScaler.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "sklearn.preprocessing.RobustScaler",
+  "contributors": ["Hector Dominguez <hedb_12@hotmail.com>"],
+  "documentation": "http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.RobustScaler.html",
+  "description": "Scale features using statistics that are robust to outliers",
+  "classifiers": {
+    "type": "preprocessor",
+    "subtype": "transformer"
+  },
+  "modalities": [],
+  "primitive": "sklearn.preprocessing.RobustScaler",
+  "fit": {
+    "method": "fit",
+    "args": [
+      {
+        "name": "X",
+        "type": "ndarray",
+        "description": "The data used to compute the median and quantiles used for later scaling along the features axis"
+      }
+    ]
+  },
+  "produce": {
+    "method": "transform",
+    "args": [
+      {
+        "name": "X",
+        "type": "ndarray"
+      }
+    ],
+    "output": [
+      {
+        "name": "X",
+        "type": "ndarray"
+      }
+    ]
+  },
+  "hyperparameters": {
+    "fixed": {
+      "with_centering": {
+        "description": "If True, center the data before scaling. This will cause transform to raise an exception when attempted on sparse matrices, because centering them entails building a dense matrix which in common use cases is likely to be too large to fit in memory",
+        "type": "bool",
+        "default": true
+      },
+      "copy": {
+        "description": "If False, try to avoid a copy and do inplace scaling instead. This is not guaranteed to always work inplace; e.g. if the data is not a NumPy array or scipy.sparse CSR matrix, a copy may still be returned",
+        "type": "bool",
+        "default": true
+      }
+    },
+    "tunable": {
+      "with_scaling": {
+        "description": "If True, scale the data to interquartile range",
+        "type": "bool",
+        "default": true
+      },
+      "quantile_range": {
```

Comment:
This is not currently supported. The only types of tunable hyperparameters that we support are `int`, `float`, `str`, `bool`, `multitype` and `conditional`: https://hdi-project.github.io/MLBlocks/advanced_usage/hyperparameters.html#tunable-hyperparameters

So, even thought this should ideally be specified as you did, for now it would be better to set this as a "fixed" hyperparameter and avoid tuning it.